requirements.txt

```diff
@@ -1,6 +1,6 @@
 datajoint>=0.13.0
-element-calcium-imaging
-element-lab
-element-animal
-element-session
-ipykernel
\ No newline at end of file
+element-calcium-imaging==0.1.0b0
```

Comment:
@kabilar Should we version lock for all Q1 2022 releases?

Reply:
Hi @CBroz1, yes, I believe it is good practice to version lock the dependencies that have been tested with our `workflow` integration tests.